src/exampleTypes/TagsText.js

```diff
@@ -52,12 +48,25 @@ let Text = ({
         tagStyle={bgJoin(tagToGroupJoin(node.join))}
         submit={tree.triggerUpdate}
         placeholder={placeholder}
+        style={{ flex: 1, border: 0 }}
       />
-      <Popover open={open}>
-        <TagsJoinPicker tag={selectedTag} node={node} tree={tree} />
-      </Popover>
-    </div>
-  )
-}
+
+      <div style={{ paddingTop: 4 }}>
+        <Popover
+          trigger={
+            <div>
+              <Icon icon="TableColumnMenu" />
+            </div>
+          }
+          position="bottom right"
+          closeOnPopoverClick={false}
+          style={{ width: 240 }}
+        >
+          <TagsJoinPicker node={node} tree={tree} />
```

Comment:
TagsJoinPicker do not need to use Tag but need to use  the node

Reply:
Yeah it doesn't use the tag prop